langwatch/src/components/evaluations/wizard/steps/evaluations/EvaluatorSettingsAccordion.tsx

```diff
@@ -93,7 +93,14 @@ export const EvaluatorSettingsAccordion = () => {
     form.reset({
       settings: defaultSettings,
     });
-    onSubmit({ settings: defaultSettings });
+
+    // Only call onSubmit if we're setting NEW default settings (not existing parameters)
+    const hasExistingParameters =
```

Comment:
The way this hook works is concerning because it's not respecting the dep array.